qgreenland/config/layers/Oceanography/seal_tag_measurements.py

```diff
@@ -0,0 +1,31 @@
+from qgreenland.config.datasets.seal_tag_water_measurements import dataset
+from qgreenland.config.helpers.steps.ogr2ogr import ogr2ogr
+from qgreenland.models.config.layer import Layer, LayerInput
+
+layer = Layer(
+    id="seal_tag_measurements",
+    title="Seal tag water measurements",
+    description=(
+        """Locaitons of seal tag measurements with temperature and salinity in coastal Greenland from 2010-08-31 to 2012-05-14."""
+    ),
+    tags=[],
+    in_package=True,
+    input=LayerInput(
+        dataset=dataset,
+        asset=dataset.assets["only"],
+    ),
+    steps=[
+        *ogr2ogr(
```

Comment:
This helper function runs `ogr2ogr` under the hood and includes default options that we normally want for all of our vector layers (e.g., reproject to our project projection). You can find the implementation for this helper here: https://github.com/nsidc/qgreenland/blob/main/qgreenland/config/helpers/steps/ogr2ogr.py#L18-L46